server/lyft/decorators/destroy_plan_project_finder_wrapper.go

```diff
@@ -0,0 +1,47 @@
+package decorators
+
+import (
+	"github.com/runatlantis/atlantis/server/events"
+	"github.com/runatlantis/atlantis/server/events/yaml/valid"
+	"github.com/runatlantis/atlantis/server/logging"
+	"path/filepath"
+	"strings"
+)
+
+type DestroyPlanProjectFinderWrapper struct {
+	events.ProjectFinder
+}
+
+func (d DestroyPlanProjectFinderWrapper) DetermineProjectsViaConfig(log logging.SimpleLogging, modifiedFiles []string, config valid.RepoCfg, absRepoDir string) ([]valid.Project, error) {
+	if len(config.Projects) == 0 {
+		return d.ProjectFinder.DetermineProjectsViaConfig(log, modifiedFiles, config, absRepoDir)
```

Comment:
As mentioned offline, we don't need this since we will be able to modify `when_modified` in our pre-workflow hook.

Reply:
done